assets/src/components/detours/pastDetourPanel.tsx

```diff
@@ -1,12 +1,87 @@
 import React from "react"
 import { Panel } from "./diversionPage"
+import { DetourDirection } from "../../models/detour"
+import { Stop } from "../../schedule"
+import { ArrowLeft } from "../../helpers/bsIcons"
+import { Button, ListGroup } from "react-bootstrap"
+import { AffectedRoute, MissedStops } from "./detourPanelComponents"
 
-export const PastDetourPanel = () => (
+export interface PastDetourPanelProps {
+  directions?: DetourDirection[]
+  connectionPoints?: string[]
```

Comment:
suggestion(non-blocking): Hmm, could we make it explicit how many points we expect?
```suggestion
  connectionPoints?: [string, string]
```

Reply:
I think you could then de-structure it out of the params if you felt like it, but I wouldn't say you _have_ to

Reply:
Huh - i didn't know you could do that.

Reply:
And since `connectionPoints` are actually always provided here, [I can destructure the value too](https://github.com/mbta/skate/pull/2817/commits/29420917b5680a1eef64aec6e3b933097e3affdd).

(I wonder if that can also be done for the `activeDetourPanel`, but that's a question for a different PR.)